aim/cli/branch/commands.py

```diff
@@ -1,9 +1,11 @@
 import click
 
+from aim.cli.branch.utils import get_branches
+
 
 @click.group()
 @click.pass_obj
-def branch_entry_point(repo):
+def branch(repo):
```

Comment:
Those entrypoints were renamed because autocompletion was using the function names as autocompletion prompts.